pointer/pointer.go

```diff
@@ -107,304 +70,218 @@ func Int32Equal(a, b *int32) bool {
 	return *a == *b
 }
 
-// Uint returns a pointer to an uint
-func Uint(i uint) *uint {
-	return &i
-}
+var (
 
-// UintPtr is a function variable referring to Uint.
-//
-// Deprecated: Use Uint instead.
-var UintPtr = Uint // for back-compat
+	// Int returns a pointer to an int
+	Int = Ref[int]
 
-// UintDeref dereferences the uint ptr and returns it if not nil, or else
-// returns def.
-func UintDeref(ptr *uint, def uint) uint {
-	if ptr != nil {
-		return *ptr
-	}
-	return def
-}
+	// IntPtr is a function variable referring to Int.
+	//
+	// Deprecated: Use Int instead.
```

Comment:
This might as well point to `Ref` directly since that’s what people updating deprecated code should end up using.